src/Common/Compat.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
[won't let me make a comment on the commit description]

can you please include a note of an example command(s) you used to generate the .cs file? will make it easier if we have to do it again